core/src/config.cpp

```diff
@@ -36,7 +36,7 @@ void ConfigManager::load(json def, bool lock) {
         file >> conf;
         file.close();
     }
-    catch (std::exception e) {
+    catch (const std::exception& e) {
```

Comment:
Please revert, just a style string that barely does anyhing for performance, I don't care much about const correcteness